src/module/actor/entity.js

```diff
@@ -282,9 +282,19 @@ export default class OseActor extends Actor {
     const actorData = this.system;
 
     const label = game.i18n.localize(`OSE.roll.hd`);
-    const rollParts = [actorData.hp.hd];
+
+    const rollParts = [];
+
     if (actorType === "character") {
-      rollParts.push(actorData.scores.con.mod * actorData.details.level);
+      const hd = `${actorData.details.level}${actorData.hp.hd.slice(
+        actorData.hp.hd.indexOf("d")
```

Comment:
errors on e.g. `2D6` which is a legal diceterm string (Foundry API lowercases it to `2d6` before rolling)

Reply:
ooh, I'll fix that up